tests/test_installed_distributions.py

```diff
@@ -65,3 +66,28 @@ def mock_call(*args, cwd=None):
     monkeypatch.setattr(pip_api._installed_distributions, "call", mock_call)
 
     _ = pip_api.installed_distributions(local=True)
+
+
+@pytest.mark.skipif(
+    pip_api.PIP_VERSION < parse("19.2"),
+    reason="Pip version too old to support --path parameter",
+)
+def test_installed_distributions_path(pip, some_distribution, target):
```

Comment:
Would be good to add a test that runs with multiple path arguments. I'll have to make some test data for a second package in the relevant `conftest.py` in order for that to work.

Reply:
8141e3f